README.md

```diff
@@ -313,6 +313,80 @@ Some Serilog packages require a reference to a logger configuration object. The
 },
 ```
 
+### Destructuring
+
+Destructuring means extracting pieces of information from an object and create properties with values; Serilog offers the [@ destructuring operator](https://github.com/serilog/serilog/wiki/Structured-Data#preserving-object-structure). In case there is a need to customize the way log events are serialized (e.g., hide property values or replace them with something else), one can define several destructuring policies, like this:
```

Comment:
We've taken to calling `@` the "structure-capturing operator", might be nicee to use that here. (Why the feature is called "destructuring" is a bit of a long story 😅)